libs/components/ag-grid/src/lib/modules/ag-grid/types/autocomplete-properties.ts

```diff
@@ -2,20 +2,63 @@ import { TemplateRef } from '@angular/core';
 import { SkyAutocompleteSelectionChange } from '@skyux/lookup';
 
 export interface SkyAgGridAutocompleteProperties {
+  /**
+   * Allows users to specify arbitrary values not in the search results.
+   * @default false
+   */
   allowAnyValue?: boolean;
+  /**
+   * The static data source for the autocomplete cell to search when users enter text. For a dynamic data source, such as an array that changes due to server calls, use search instead. You can specify static data, such as an array of objects, or you can pull data from a database.
+   */
   data?: unknown[];
+  /**
+   * How many milliseconds to wait before searching while users enter text in the autocomplete field.
+   * @default 0
+   */
   debounceTime?: number;
+  /**
+   * The object property to display in the text input after users select an item in the dropdown list.
+   * @default "name"
+   */
   descriptorProperty?: string;
+  /**
+   * Highlights the search text in each search result. Set this to `false` when your search returns results that aren't exact text matches, such as returning "Bob" for "Robert."
+   * @default true
+   */
   highlightSearchText?: boolean;
+  /**
+   * The array of object properties to search when utilizing the `data` property and the built-in search function.
+
+   * @default ["name"]
+   */
   propertiesToSearch?: string[];
+  /**
+   * The function that dynamically manages the data source when users change the text in the autocomplete cell. The search function must return an array or a promise of an array.
+   */
   search?: (
     searchText: string,
     data?: unknown[],
   ) => unknown[] | Promise<unknown[]>;
+  /**
+   * The array of functions to call against each search result. This filters the search results when using the default search function. When the `search` property specifies a custom search function, you must manually apply filters inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
```

Comment:
```suggestion
   * The array of functions to call against each search result. This filters the search results when using the `data` input and the default search function. When the `search` property specifies a custom search function, you must manually apply filters inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
```
The corresponding description for lookup says "when using the data input and the default search function" instead of just "when using the default search function." Should that be the same here too? It looks like both components have a `data` property.